docs/annotation-cheat-sheet.md

```diff
@@ -0,0 +1,161 @@
+# Annotation Cheat-Sheet
+
+How to construct an annotation from a term definition, or re-engineer a term definition from an annotation example?
+
+Here's how.
+
+## Vocabularies
+
+Terms are defined within a "vocabulary", which is just an EDMX document.
+
+```xml
+<edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
+  <edmx:DataServices>
+
+    <Schema Namespace="Vocab"
+
+      xmlns="http://docs.oasis-open.org/odata/ns/edm">
+```
+
+To use a term in an annotation, the vocabulary of the term needs to be referenced and included.
+
+```xml
+<edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
+  <edmx:Reference Uri="https://somewhere.org/vocab.xml">
+
+    <edmx:Include Namespace="Vocab" />
+
+  </edmx:Reference>
+```
+
+## Primitive Terms
+
+If the term has a primitive type
+
+```xml
+<Term Name="StringTerm" Type="Edm.String" />
+```
+
+A constant annotation value is provided in a [corresponding attribute](https://docs.oasis-open.org/odata/odata-csdl-xml/v4.01/odata-csdl-xml-v4.01.html#sec_ConstantExpression).
+
+```xml
+<Annotation Term="Vocab.StringTerm" String="annotation value" />
+```
+
+A dynamic annotation value can be provided for the same term using a [value path expression](https://docs.oasis-open.org/odata/odata-csdl-xml/v4.01/odata-csdl-xml-v4.01.html#sec_ValuePath).
+
+```xml
+<Annotation Term="Vocab.StringTerm" Path="SomeStringProperty" />
+```
+
+The property referenced via the value path expression needs to have the same type as the term. A value path expression can always be used instead of a constant value, also in the more complicated cases below.
+
+## Collections
+
+If the term has a collection type
+
+```xml
+<Term Name="CollectionTerm" Type="Collection(Edm.Decimal)" />
+```
+
+the annotation value is provided as a [collection expression](https://docs.oasis-open.org/odata/odata-csdl-xml/v4.01/odata-csdl-xml-v4.01.html#sec_Collection)
+
+```xml
+<Annotation Term="Vocab.CollectionTerm">
+  <Collection>
+    <Decimal>2.78</Decimal>
+    <Decimal>3.14</Decimal>
+  </Collection>
+</Annotation>
+```
+
+## Structures
+
+If the term has a structured type
+
+```xml
+<Term Name="StructuredTerm" Type="Vocab.Complex" />
+
+<ComplexType Name="Complex">
+  <Property Name="IntegerField" Type="Edm.Int32" />
+</ComplexType>
+```
+
+the annotation value is provided as a [record expression](https://docs.oasis-open.org/odata/odata-csdl-xml/v4.01/odata-csdl-xml-v4.01.html#sec_Record)
+
+```xml
+<Annotation Term="Vocab.StructuredTerm">
+  <Record>
```

Comment:
Please mention that a Record can have a Type attribute, and when this must be specified.